src/store/gateway/state.js

```diff
@@ -119,5 +119,9 @@ export default {
     bans: [],
     tx_pool_backlog: [],
     service_nodes: []
+  },
+  update_required: {
+    required: false,
+    latest_version: "Unknown"
```

Comment:
seems we're not using `latest_version` so we can just make `update_required a boolean value

Reply:
Yeah was more thinking forward on this one, that it could be useful to store.